### PR TITLE
mvp scope init

### PR DESCRIPTION
**Pull request by marcusellison**
Adapted from utb 001 **User Experience Questions** discussion

### CODE REVIEW

File: bdr/009-mvp-scope.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# MVP SCOPE
+
+We wanted to narrowly define our intended experience for our MVP.
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:3]: This is just context for me. Also, let's get the link to the UTB in here to cross reference.

File: bdr/009-mvp-scope.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### Context
+
+We have an ideal experience we intend to build but need to first build a product quickly that we can get feedback on and iterate.
+
+We need to choose which features to build first.
+
+
+
+### Alternatives
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:15]: I started to put the decisions first and then the alternatives. I found this reads more naturally.

File: bdr/009-mvp-scope.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+* Focus only on metric updates
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:17]: For now, this is not an alternative, this is what we want to do. Would you agree?

File: bdr/009-mvp-scope.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+* Allow for multiple update types
+* Build in privacy considerations
+* Build in a dispatch system
+
+
+
+### Decisions
+
+- The core user experience is a single timeline of metric updates.
+- The user simply writes in the action bar. And can can an update in 1-click
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:27]:
```suggestion
- The user simply writes in the action bar. And can create an update with just 1-click
```

File: bdr/009-mvp-scope.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- We will focus on 4 cool features
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:28]: With the editor and the data point links it is 6.

File: bdr/009-mvp-scope.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  - flip update to reveal graph
+  - discover where update is in timeline
+  - placeholder design for hashtags and mentions
+  - publicly shareable url
+
+
+### Consequences
+
+We believe this decision shortens build timeline while offering the minimum viable experience to get to our vision.
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:37]:
```suggestion
We believe this decision shortens build time while offering the minimum viable experience to get to our vision.
```

File: bdr/009-mvp-scope.md (hunk 3 above)
Comment (marcusellison) [reply bdr/009-mvp-scope.md:17]: I was trying to portray the alternatives as between prioritizing vs larger scope.

File: bdr/009-mvp-scope.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+We need to choose which features to build first.
+
+(brainstorm link)[https://github.com/venturemark/whiteboard/tree/master/utb]
+
+
+### Decisions
+
+- The core user experience is a single timeline of metric updates.
+- The user simply writes in the action bar. And can create an update with just 1-click
+- We will focus on 4 cool features
```
Comment (xh3b4sd) [inline bdr/009-mvp-scope.md:19]:
```suggestion
- We will focus on 6 cool features
```

**Review by xh3b4sd — APPROVED** [review]

**Pull request merged**